lib/dry/system/plugins/dependency_graph.rb

```diff
@@ -20,7 +20,7 @@ def self.extended(system)
             self[:notifications].register_event(:resolved_dependency)
             self[:notifications].register_event(:registered_dependency)
 
-            system.strategies(Stratagies.with_notifications(system[:notifications]))
+            strategies(Stratagies)
```

Comment:
Is that a typo? If so, it's also in the `lib/dry/system/plugins/dependency_graph/strategies.rb`

```suggestion
            strategies(Strategies)
```

Reply:
hey, yep. My bad. thanks for finding this